### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [nicolargo/glances#2216] [Q&A] [answered]
Hi all.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by nicolargo** [comment, marked as answer]
Yes for sure.
Have a look to the issue tracter.
Look for Need contribution tag.

Nicolas


Le sam. 17 déc. 2022, 20:13, Code Hangen ***@***.***> a
écrit :

**Reply by ghost** [reply]
Hi nicolargo, thank you for feedback.
